docs/dataflow.rst

```diff
@@ -2,21 +2,21 @@
 Data flow from submission to results
 ------------------------------------
 
-The graph below summarize the EBI Metagenomics data flow from submission to :term:`analysis results<analysis result>`:
+The graph below summarize the MGnify data flow from submission to :term:`analysis results<analysis result>`:
 
 .. figure:: images/submit_graph_08_web032.png
    :scale: 50 %
 .. https://stackoverflow.com/questions/12297493/why-does-image-scale-not-work-in-restructuredtext-when-generating-html-files
 
 **(1) Submissions are handled by the** `European Nucleotide Archive (ENA) <https://www.ebi.ac.uk/ena/>`_ **and therefore users have to have an ENA** `Webin account <https://www.ebi.ac.uk/ena/submit/sra/>`_.
 
-   In addition, users submitting private data have to provide an expressed agreement that EBI Metagenomics can access their data for analysis, as described under `Submit data <https://www.ebi.ac.uk/metagenomics/submission>`_. Otherwise, we will not be able to access their data. EBI Metagenomics will, of course, handle these data confidentially.
+   In addition, users submitting private data have to provide an expressed agreement that MGnify can access their data for analysis, as described under `Submit data <https://www.ebi.ac.uk/metagenomics/submission>`_. Otherwise, we will not be able to access their data. MGnify will, of course, handle these data confidentially.
```

Comment:
more a question than a comment: due to GDPR, could we assume that users who gave EBI Metagenomics access agreement have done so too to Mgnify?